snapcraft/internal/pluginhandler/_patchelf.py

```diff
@@ -150,12 +140,17 @@ def patch(self) -> None:
         :raises errors.SnapcraftEnvironementError:
             if something is horribly wrong.
         """
-        # Just return if this is a static base and libc6 has not been staged.
-        if self._project.is_static_base(self._core_base) and not self._is_libc6_staged:
+        # Just return if this is None or a static base and libc6 has not been staged.
```

Comment:
'this is None' -> base is None?

maybe collapse the two if statements to make the comment work better?